### PR TITLE
Added differential drive capability

### PR DESCRIPTION
**Pull request by FlareTheSaberDragon**
This Pull Request closes issue #

Description of the changes:
 - 

 - [ ] I have labelled this issue with the priority and correct subsystem.
 - [ ] I set the person that I worked on this with as a reviewer (who will need to approve the changes)

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -21,7 +20,10 @@
 public class RobotContainer {
   // Subsystems
   public final ManipulatorSubsystem manipulator = new ManipulatorSubsystem();
-  public final MecanumDrive drive = new MecanumDrive();
+
+  // one of the two following lines of code must be commented out at all times
+  public final DiferentialDrive drive = new DiferentialDrive();
```
Comment (LinusSkucas) [inline src/main/java/frc/robot/RobotContainer.java:25]: Let's just switch to using Mecanum for now

File: src/main/java/frc/robot/subsystems/DiferentialDrive.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
+package frc.robot.subsystems;
```
Comment (LinusSkucas) [inline src/main/java/frc/robot/subsystems/DiferentialDrive.java:1]: This file is unneeded (it's just a spelling thing)

File: src/main/java/frc/robot/subsystems/DifferentialDrive.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+  @Override
+  public void periodic() {
+    // This method will be called once per scheduler run
+    super.periodic();
+
+    SmartDashboard.putBoolean("isInverted", inverted);
+  }
+
+  public void updateSpeed(double strafe, double drive, double turn, boolean useInverted) {
```
Comment (LinusSkucas) [inline src/main/java/frc/robot/subsystems/DifferentialDrive.java:77]:
```suggestion
  // Use strafe parameter just to be backwards compatible
  public void updateSpeed(double strafe, double drive, double turn, boolean useInverted) {
```

**Review by LinusSkucas — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/DiferentialDrive.java (hunk 2 above)
Comment (FlareTheSaberDragon) [reply src/main/java/frc/robot/subsystems/DiferentialDrive.java]: deleted!

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (FlareTheSaberDragon) [reply src/main/java/frc/robot/RobotContainer.java]: switched to mecanum

File: src/main/java/frc/robot/subsystems/DifferentialDrive.java (hunk 3 above)
Comment (FlareTheSaberDragon) [reply src/main/java/frc/robot/subsystems/DifferentialDrive.java:69]: added the clarifying comment

**Review by LinusSkucas — APPROVED** [review]

**Review by SBsteveyHops — APPROVED** [review]

**Pull request merged**